### ISSUE TITLE
MDCameraMockup test XML contains path to specific jpeg file

### ISSUE THREAD

**Issue by mguijarr** [mxcube/mxcube3#78]
The `md_camera.xml` file that is shipped with MXCuBE 3 in `test/HardwareObjectsMockup.xml` directory specifies a path to an undistributed jpeg file:

```
<device class="MDCameraMockup">
  [...]
  <image_path>/mxn/home/mikegu/mxcube_sample_snapshot.jpeg</image_path>
</device>
```

It would be great if the sample snapshot jpeg file could be shipped with MXCuBE 3, and if the path could be fixed, so everybody can test MXCuBE 3 without changing test XML files.

**Comment by meguiraun** [comment]
facepalm! I am on it...

**Comment by mguijarr** [comment]
Solved by PR #79
